verilog/tools/ls/README.md

```diff
@@ -236,3 +271,55 @@ code --install-extension verible.vsix
 ```
 
 [release]: https://github.com/chipsalliance/verible/releases
+
+## Configuring the Language Server for a project
```

Comment:
I think it would be good to have these above the editor configuration, as it is probably the information that people most come back to: they only set up their editor once, but they might need to re-read what the name of the file list is etc.